diff --git a/node_modules/react-native-waveform/android/build.gradle b/node_modules/react-native-waveform/android/build.gradle
index 5fcf1c6..abdf62e 100644
--- a/node_modules/react-native-waveform/android/build.gradle
+++ b/node_modules/react-native-waveform/android/build.gradle
@@ -1,25 +1,27 @@
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
     }
-
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.+'
+        classpath 'com.android.tools.build:gradle:8.0.0'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    namespace "com.reactnativewaveform"   // required for modern Gradle
+
+    compileSdkVersion 36
+    buildToolsVersion "36.0.0"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion 24
+        targetSdkVersion 36
         versionCode 1
         versionName "1.0"
     }
+
     lintOptions {
         abortOnError false
     }
@@ -30,5 +32,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.18.+'
+    implementation 'com.facebook.react:react-native:0.18.+'
 }
\ No newline at end of file
